common/ledger/blkstorage/snapshot_test.go

```diff
@@ -123,6 +123,18 @@ func TestImportFromSnapshot(t *testing.T) {
 		require.NoError(t, err)
 		bootstrappedBlockStore, err = env.provider.Open(bootstrappedLedgerName)
 		require.NoError(t, err)
+
+		isFromSnapshot, err := IsBootstrappedFromSnapshot(env.provider.conf.blockStorageDir, "originalLedger")
+		require.NoError(t, err)
+		require.False(t, isFromSnapshot)
+
+		isFromSnapshot, err = IsBootstrappedFromSnapshot(env.provider.conf.blockStorageDir, bootstrappedLedgerName)
+		require.NoError(t, err)
+		require.True(t, isFromSnapshot)
+
+		ledgersFromSnapshot, err := GetLedgersBootstrappedFromSnapshot(env.provider.conf.blockStorageDir)
+		require.NoError(t, err)
+		require.ElementsMatch(t, ledgersFromSnapshot, []string{bootstrappedLedgerName})
 	}
```

Comment:
This seems wrongly placed here in the `setup` function of the test `TestImportFromSnapshot`. You can have a separate UT for this and for this, you can just create a few folders with `bootstrappingSnapshotInfoFile` file under a few of those.

Reply:
Moved tests to  blockfile_helper_test.go